tests/integration/testblockexpiration.nim

```diff
@@ -44,19 +44,21 @@ ethersuite "Node block expiration tests":
     client.close()
     uploadResponse.body
 
-  proc downloadTestFile(contentId: string): Response =
+  proc downloadTestFile(contentId: string, local = false): Response =
     let client = newHttpClient(timeout=3000)
-    let downloadUrl = baseurl & "/data/" & contentId
+    let downloadUrl = baseurl & "/data/" &
+      contentId & (if local: "" else: "/network")
+
     let content = client.get(downloadUrl)
     client.close()
     content
 
   proc hasFile(contentId: string): bool =
     let client = newHttpClient(timeout=3000)
-    let dataLocalUrl = baseurl & "/local"
+    let dataLocalUrl = baseurl & "/data/" & contentId
```

Comment:
I think this proc was intended to check the local-content-listing API. Downloading the file sure checks that the file is there. But I suppose we should have another check then to see that the content-listing API is OK.

Reply:
Yeah, that makes sense.

We might also further refine the api by making `/data/{cid}` return a `200/404` response codes to check for presence, and then have dedicated `/data/{cid}/network` and `/data/{cid}/local` to fetch the file. I'm not that concerned with the namespasing as long as it's descriptive.

Reply:
oh, btw - in this particular case, there should have been an explicit test to check the `listing` api endpoint, I can add it.

Reply:
I've added a dedicated test for the `listing` endpoint.